server/services/locationService.test.ts

```diff
@@ -8,13 +8,15 @@ describe('location service', () => {
   let fakeDataPlatformApi: nock.Scope
   const dummyData: Location[] = [
     {
-      latitude: '20.0',
-      longitude: '20.0',
+      id: 1,
+      latitude: 20.0,
+      longitude: 20.0,
       locationTime: '2023:12:12T12:34:00',
     },
     {
-      latitude: '20.0',
-      longitude: '20.0',
+      id: 2,
+      latitude: 20.0,
+      longitude: 20.0,
```

Comment:
Remove fake data or implement a dummy service for testing that can be injected to the controllers.